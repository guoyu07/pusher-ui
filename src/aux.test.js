import React from 'react';
import renderer from 'react-test-renderer';

import Aux from './aux';

describe('Aux', () => {
  it('should render children without wrapping them with an element', () => {
    expect(
      renderer
        .create(
          <Aux>
            <div>hello</div>
          </Aux>,
        )
        .toJSON(),
    ).toMatchSnapshot();

    expect(
      renderer
        .create(
          <Aux>
            <div>one</div>
            <div>two</div>
            <div>three</div>
          </Aux>,
        )
        .toJSON(),
    ).toMatchSnapshot();

    expect(
      renderer.create(<Aux>Very interesting!</Aux>).toJSON(),
    ).toMatchSnapshot();
  });
});
