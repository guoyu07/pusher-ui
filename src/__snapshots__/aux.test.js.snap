// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Aux should render children without wrapping them with an element 1`] = `
<div>
  hello
</div>
`;

exports[`Aux should render children without wrapping them with an element 2`] = `
Array [
  <div>
    one
  </div>,
  <div>
    two
  </div>,
  <div>
    three
  </div>,
]
`;

exports[`Aux should render children without wrapping them with an element 3`] = `"Very interesting!"`;
